src/app/shared/components/notification-card/notification-card.component.ts

```diff
@@ -29,30 +20,14 @@ export class NotificationCardComponent implements OnInit {
   public readonly Icon = Icon
   public readonly IconSize = IconSize
 
-  constructor(private readonly elementRef: ElementRef) {}
+  constructor() {}
```

Comment:
The `constructor` can be removed. If it is not removed, then it should have a access modifier :)

Reply:
Done